<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaultForm</class>
 <widget class="QWidget" name="FaultForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>45</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FaultForm{
border:2px solid #888888;
}
QPushButton#btn_del{
 
	background-color: rgb(78, 154, 6);
   color:white;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>spn:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_spn">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>fmi:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_fmi">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>model:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_model">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>counter:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_counter">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_status">
     <property name="text">
      <string>status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_del">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>172</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
